gusto/equations/common_forms.py

```diff
@@ -172,6 +182,34 @@ def vector_invariant_form(domain, test, q, ubar):
     return transport(form, TransportEquationType.vector_invariant)
 
 
+def linear_vector_invariant_form(domain, test, q, ubar):
+    u"""
+    The linear form corresponding to the vector invariant transport operator.
+
+    The vector invariant transport operator is: (∇×q)×u + (1/2)∇(u.q)
+    and its linearised form is:
+    (∇×q')×u_bar + (∇×q_bar)×u' + (1/2)∇(u_bar.q') + (1/2)∇(u'.q_bar)
+
+    Args:
+        domain (:class:`Domain`): the model's domain object, containing the
+            mesh and the compatible function spaces.
+        test (:class:`TestFunction`): the test function.
+        q (:class:`ufl.Expr`): the variable to be transported.
+        ubar (:class:`ufl.Expr`): the transporting velocity.
+
+    Returns:
+        class:`LabelledForm`: a labelled transport form.
+    """
+
+    L = linear_circulation_form(domain, test, q, ubar).form
```

Comment:
I'm beginning to think q must be qbar + q' ... maybe in the docs for the functions above make it clear?